README.md

```diff
@@ -52,12 +53,34 @@ Note that the auspice development mode uses the auspice splash page which won't
 This is because the nextstrain.org server (see above) uses the splash page of the static content, but the auspice dev server doesn't.
 The advantage of development mode is that the client will live update as you edit the customisations.
 
-
+---
 ## Server
-`./server.js` decides, based on the path, whether to serve auspice or the (pre-built) static documentation / splash pages.
-It also imports auspice-specific code from `./auspice/server` to process auspice requests for data (including data transforms).
+Auspice is a flexible tool created for visualising phylogenomic data that are not specific to any domain.
+Therefore, auspice installations do not come with any pre-packaged data.
+Auspice requires interaction with a server to send and receive data.
+The [auspice repo on GitHub](http://github.com/nextstrain/auspice) contains a NodeJS backend that serves the bundled auspice JavaScript file to the server and also provides web API endpoints (prefixed with '`/charon`).
+The auspice front-end (e.g. "client") is server-agnostic. Currently, however, the auspice backend (e.g. "server") is not implemented in an importable or flexible way that allows usage across servers.
+This means that nextstrain.org, auspice.us, and your local auspice installation all use different servers.
 
+[./server.js](server.js) decides, based on the path, whether to serve the bundled auspice JavaScript file (built from running `auspice build`), the (pre-built) static [splash & documentation pages](#Splash--documentation-pages), or the auspice-specific code from `./auspice/server` to process auspice requests for data from the `/charon` web API endpoints (including data transforms).
 
+---
+## Build nextstrain.org locally
+
+1. `npm install` (from the "nextstrain.org" directory)
+2. `npm run set-up`, which runs `./set-up.sh` to build both the static site & auspice.
+3. `npm run server` will then start a local instance, by default available at [localhost:5000](http://localhost:5000).
+
+Because the nextstrain.org server and the auspice backend are completely independent, you cannot currently run `auspice view` (with hot reloading) to serve the nextstrain.org server.
```

Comment:
some clarifications:

*  They're independent in the sense that they're both "servers" for an auspice client -- i.e. they both serve the auspice client JS code, and listen for GET requests (`/charon/...`) -- so it doesn't make sense to try to combine them. 

* `auspice view` doesn't have hot reloading - that's `auspice develop`.

* The actual code that handles `/charon/...` requests in nextstrain.org (see https://github.com/nextstrain/nextstrain.org/blob/master/auspice/server/index.js) has been written in such a way that that it can be imported by 
    (i) the nextstrain.org server
    (ii) `auspice view --handlers ./server/index.js`
    (iii) `auspice develop --handlers ./server/index.js`